explorer/interpreter/interpreter.cpp

```diff
@@ -1298,7 +1299,7 @@ auto Interpreter::StepExp() -> ErrorOr<Success> {
           CARBON_ASSIGN_OR_RETURN(
               Nonnull<const Value*> string_value,
               Convert(args[1], arena_->New<StringType>(), exp.source_loc()));
-          if (cast<BoolValue>(condition)->value() == false) {
+          if (!cast<BoolValue>(condition)->value()) {
```

Comment:
I think this change makes the code less readable: written this way, it's not clear what kind of truthiness test is being performed. As a reader, I'm thinking that maybe `value` returns an `optional`. To silence clang-tidy, how about this:
```suggestion
          bool condition_value = cast<BoolValue>(condition)->value();
          if (!condition_value) {
```